src/common.c

```diff
@@ -185,6 +185,9 @@ int ofi_ep_bind_valid(struct fi_provider *prov, struct fid *bfid, uint64_t flags
 			return -FI_EBADFLAGS;
 		}
 		break;
+	case FI_CLASS_SRX_CTX:
+		/* Flags can be ignored */
+		break;
```

Comment:
Currently no flags are defined for this binding, but we may add flags in the future. For better future proof, I am more inclined to say right now flags must be 0 here.